backend/dataall/modules/metadata_forms/services/metadata_form_service.py

```diff
@@ -0,0 +1,74 @@
+from dataall.base.context import get_context
+from dataall.base.db import exceptions, paginate
+from dataall.core.organizations.db.organization_repositories import OrganizationRepository
+from dataall.core.environment.db.environment_repositories import EnvironmentRepository
+
+from dataall.modules.metadata_forms.db.enums import MetadataFormVisibility
+from dataall.modules.metadata_forms.db.metadata_form_repository import MetadataFormRepository
+
+
+class MetadataFormParamValidationService:
+    @staticmethod
+    def validate_create_form_params(data):
+        visibility = data.get('visibility', MetadataFormVisibility.Team.value)
+        if not MetadataFormVisibility.has_value(visibility):
+            data['visibility'] = MetadataFormVisibility.Team.value
+
+        if 'SamlGroupName' not in data:
+            raise exceptions.RequiredParameter('SamlGroupName')
```

Comment:
rather than check if the key exists in data I think we need to check the value is not null or empty string - otherwise I can still create a metadata from with empty string value

Reply:
I believe same for homeEntity need to check value as we always will pass the key from FE

Reply:
[aea1745](https://github.com/data-dot-all/dataall/pull/1444/commits/aea1745ee8adcd027cb019f967e21c5c3a4849ca)